app/controllers/events_controller.rb

```diff
@@ -0,0 +1,11 @@
+class EventsController < ApplicationController
+
+  def index
+    @events = Event.all
+  end
+
+  def show
+    @events =Event.find(params[:id])
+   end
+
+end
```

Comment:
I'm not sure why it's still in there I wanted the whole file removed for this pull. Must be the git issues will fix. It was in there by mistake.